implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/DynatraceConfigTest.java

```diff
@@ -62,4 +63,32 @@ void valid() {
 
         assertThat(config.validate().isValid()).isTrue();
     }
+
+    @Test
+    void invalidVersion() {
+        Map<String, String> properties = new HashMap<String, String>() {{
+            put("dynatrace.apiToken", "secret");
+            put("dynatrace.uri", "https://uri.dynatrace.com");
+            put("dynatrace.deviceId", "device");
+            put("dynatrace.apiVersion", "v-INVALID");
+        }};
+
+        DynatraceConfig config = properties::get;
+
+        assertThrows(IllegalArgumentException.class, config::validate);
```

Comment:
Please always check the error message and the cause too.

Reply:
The cause seems to be `null` for this case. As far as I understand that means that the Exception is already the root cause, does that seem right to you?